web/package.json

```diff
@@ -53,6 +54,7 @@
     "focus-trap-react": "8.8.2",
     "i18next": "^20.6.1",
     "lodash": "^4.17.21",
+    "mapbox-gl": "npm:empty-npm-package@1.0.0",
```

Comment:
has to be added to avoid empty packages for peer deps
https://visgl.github.io/react-map-gl/docs/get-started/get-started#using-with-a-mapbox-gl-fork